lib/mimdb_web/templates/actor/index.html.eex

```diff
@@ -0,0 +1,30 @@
+<h1>Listing Actors</h1>
+
+<table>
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th>Birthdate</th>
+      <th>Age</th>
+
+      <th></th>
+    </tr>
+  </thead>
+  <tbody>
+<%= for actor <- @actors do %>
+    <tr>
+      <td><%= actor.name %></td>
+      <td><%= actor.birthdate %></td>
+      <td><%= age(&Date.utc_today/0, actor) %></td>
```

Comment:
I think this is a bug. UTC will not alway correspond to the date in the user time zone. Might be a very narrow edge case but should be aware of such considerations. Consider using Date.today("timezone_name_inserted_here") with tzdata.

Reply:
Sorry should be something like:
 DateTime.now!("America/Denver")
    |> DateTime.to_date()

only with a more logical tz

Reply:
Thank you @mratliff for this! I've added a new comment implementing your reviews!